ApptentiveFeedback/ApptentiveFeedback/Additions/ApptentiveConfiguration.cs

```diff
@@ -18,11 +18,10 @@ public class ApptentiveConfiguration
         public bool ShouldInheritAppTheme { get; set; } = true;
         public bool ShouldSanitizeLogMessages { get; set; } = true;  
         public string CustomAppStoreURL { get; set; } = null;
-        public string DistributionName { get; set; } = "";
-        public string DistributionVersion { get; set; } = "";
+        private string DistributionName { get; } = "Xamarin";
+        private string DistributionVersion { get; } = "";
```

Comment:
Is this where we'll set the version to 6.0.4 or is there another area?

Reply:
I fixed it by setting it 6.0.4